elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/SQLQueryEngine.java

```diff
@@ -441,6 +446,7 @@ private String extractProjection(Query query) {
                 .collect(Collectors.toList());
 
         List<String> dimensionProjections = query.getDimensions().stream()
+                .map(requestedDim -> query.getSchema().getDimension(requestedDim.getName()))
```

Comment:
Line 448-452 and 471-475 are the same, should we make a method for it?

Reply:
👍